src/transformers/models/t5/modeling_tf_t5.py

```diff
@@ -268,7 +268,7 @@ def _relative_position_bucket(relative_position, bidirectional=True, num_buckets
         max_exact = num_buckets // 2
         is_small = tf.math.less(relative_position, max_exact)
         relative_position_if_large = max_exact + tf.cast(
-            tf.math.log(relative_position / max_exact)
+            tf.math.log(tf.cast(relative_position, tf.float32) / tf.cast(max_exact, tf.float32))
```

Comment:
Maybe cast `relative_position` a few lines above is cleaner?

Reply:
I thought about it but the code would become more complicated (lines 274 and 277 use non-casted relative_position).